src/Microsoft.Health.Fhir.Core/Features/Operations/ConvertData/ContainerRegistryTemplateProvider.cs

```diff
@@ -28,6 +29,8 @@ public class ContainerRegistryTemplateProvider : IConvertDataTemplateProvider, I
         private readonly ITemplateCollectionProviderFactory _templateCollectionProviderFactory;
         private readonly ConvertDataConfiguration _convertDataConfig;
         private readonly MemoryCache _cache;
+        private MemoryCache _templateProviderCache;
```

Comment:
Should we mark these private attributes are readonly? As I see, they are only assigned in the constructor.

Reply:
yes thanks for the suggestion, I've updated these attributes to be readonly.